src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/ConfigurationGeneral.xaml

```diff
@@ -25,6 +25,11 @@
             <DataSource Persistence="ProjectFile" PersistedName="TargetFrameworkIdentifier" HasConfigurationCondition="False" />
         </StringProperty.DataSource>
     </StringProperty>
+    <StringProperty Name="TargetFrameworks" DisplayName="Target Frameworks">
```

Comment:
This isn't strictly necessary right? Since you're reading directly from msbuild. However Balaji is going to need this soon - so it's fine to leave it as is.